pom.xml

```diff
@@ -686,48 +687,19 @@ flexible messaging model and an intuitive client API.</description>
         <artifactId>apache-rat-plugin</artifactId>
         <configuration>
           <excludes>
-            <exclude>**/target/**</exclude>
-            <exclude>data/**</exclude>
-            <exclude>logs/**</exclude>
-            <exclude>**/test-output/**</exclude>
-            <exclude>**/*.hgrm</exclude>
-            <exclude>**/*.test</exclude>
-
             <!-- Other license files -->
             <exclude>licenses/LICENSE-*.txt</exclude>
             <exclude>src/assemble/README.bin.txt</exclude>
-
-            <!-- Exclude site files -->
-            <exclude>site/Gemfile.lock</exclude>
-            <exclude>site/VERSIONS</exclude>
-            <exclude>site/_sass/bootstrap-sass/**</exclude>
-            <exclude>site/_sass/font-awesome/**</exclude>
-            <exclude>site/.sass-cache/**</exclude>
-            <exclude>site/fonts/**</exclude>
-            <exclude>site/img/**</exclude>
-            <exclude>site/js/jquery.scrollTo.min.js</exclude>
-            <exclude>site/js/jquery.tocify.min.js</exclude>
-            <exclude>generated-site/**</exclude>
-
-            <exclude>**/*.patch</exclude>
-            <exclude>.github/*.md</exclude>
-
             <exclude>src/assemble/LICENSE.bin.txt</exclude>
             <exclude>src/assemble/NOTICE.bin.txt</exclude>
 
             <!-- Protobuf generated files -->
-            <exclude>**/proto/*.java</exclude>
+            <exclude>src/main/java/org/apache/bookkeeper/mledger/proto/MLDataFormats.java</exclude>
```

Comment:
formatting?

Reply:
Perhaps a comment can be added to explain these excludes?

Reply:
Sure, I'll expand the comment to be a bit more clear.

@rdhabalia What do you mean by formatting? The indentation looks ok to me.

Reply:
> The indentation looks ok to me.

Yes, I think in review-window, indentation didn't appear correctly. But it seems correct. So, please ignore it.